com.github.imagefanreloaded.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "com.github.imagefanreloaded",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "22.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "ImageFanReloaded-Avalonia",
+    "finish-args" : [
+		"--socket=x11",
+        "--share=ipc",
+        "--device=dri",
+		"--filesystem=host:ro"
```

Comment:
host is a very broad filesystem permission

Reply:
Thank you for the feedback!

As the application is an image viewer, it needs permissions to all folders that potentially contain images. The sensitive system folders are already excluded by flatpak, according to the official documentation, and the access that I need is read-only.

If you think it a better idea to only require read-only access on /home, /media and /mnt, please let me know.

Reply:
I've decided to narrow down the permissions, seeing that the flatpak application functions well with a restricted permission set.